test/integration/test-amp-inabox.js

```diff
@@ -212,6 +212,7 @@ describe.configure().skipWindows().run('inabox', function() {
     });
   });
 
+  // TODO(zombifier, #21311): Investigate why Safari on SauceLabs timeout.
```

Comment:
@zombifier This test occasionally times out on Chrome too: https://travis-ci.org/ampproject/amphtml/jobs/524635005#L1527

Reply:
And for single-pass builds: https://travis-ci.org/ampproject/amphtml/jobs/524681807#L1908-L1912